### ISSUE TITLE
[feature]: Moving Issues between projects

### ISSUE THREAD

**Issue by Flo4604** [makeplane/plane#1599]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

I would like to move Issue A In Project X into Project Y 

### Why should this be worked on?

This would help us have a general backlog in where we store stuff that we can check beforehand if it's something we would like to do, and once that is done move into their respective Project. 

For this making Issues ID's unique per workspace instead of project would probably help

**Comment by rhea0110** [comment]
Hey @Flo4604! Thank you for posting the feature request. We will start working on this soon. Till then, stay tuned with us!

**Comment by Plorenzo** [comment]
At least a few times every day, I create a task in an incorrect project, being able to move it right away would save so much time, instead now I have to manually copy all fields, go to the other project and create a new task, then go back and delete the incorrect task

**Comment by kotasudhakar** [comment]
The are worst 10 different issues created and yet open requesting for this feature

**Comment by emmgfx** [comment]
No news about this?

**Comment by barnabynagy** [comment]
How can I move an issue from one project to another???

**Comment by kotasudhakar** [comment]
Team, someone please help with this feature

**Comment by alekwo** [comment]
This will be very useful

**Comment by leedsjb** [comment]
@theparthacus recommend archiving this issue in favor of this one: https://github.com/makeplane/plane/issues/2733

**Comment by vihar** [comment]
Sorry this went unanswered for so long, especially given how many people asked for it.

Moving a work item between projects is built — it's Work Item Transfers, on Pro and above.

To be straight about the community edition: there you can move a workspace draft into a project, but not an item from one project to another.

Closing this out.
